README.md

```diff
@@ -1,51 +1,34 @@
+## Contact Us
 
-## Purpose (Why)
+Want to work with us? Get in touch - <a href="mailto:hello@rootsystems.nz">hello@rootsystems.nz</a>
 
-Our purpose is to imagine and cultivate local replicable life-support systems for interdependent abundance.
+## Contents
 
-## Strategy (How)
-
-We primarily do tech consulting to bootstrap our more aspirational outcomes.
-
-Want to work with us? Here's an example of our _ideal_ external consulting contract:
-
-- You, the client, have an existing source of revenue and want to build a new or replace an old software product or service.
-- We start with a series of collaborative discovery workshops to focus and clarify the problem we're solving with you.
-- We connect with relevant stakeholders to refine the scope into clear and actionable [job stories](https://jtbd.info/replacing-the-user-story-with-the-job-story-af7cdee10c27#.layvu3kpi), preferably with accompanying [executable specifications](http://www.agilemodeling.com/essays/executableSpecifications.htm).
-- We begin our iterative development process, charging hourly rates as fuel to push the vehicle forward.
-- We continue to check back in with stakeholders at each relevant cycle, to ensure our work matches the needs of your business.
-
-Meanwhile, internally we spend our available energy to:
-
-- Improve the toolbelt we carry with us in our work (our open source stack, e.g. [`cat-stack`](https://github.com/enspiral-root-systems/cat-stack), [`dogstack`](https://github.com/iainkirkpatrick/dogstack), [`inu`](https://github.com/ahdinosaur/inu), [`vas`](https://github.com/ahdinosaur/vas), and more)
-- Plan our economic needs, wants, and offers together (i.e. selling services, saving money, buying food), to ensure sustainable livelihood for all our members.
-- Cultivate ecosystems of interoperable life-support products, [platforms](http://platformcoop.net/), and protocols (e.g. [Open App Ecosystem](https://github.com/open-app/core) -> [Collaborative Technology Alliance](https://medium.com/enspiral-tales/doing-more-together-together-seeding-a-collaborative-technology-alliance-82243ea30d41), [Secure Scuttlebutt Consortium](https://www.scuttlebutt.nz), [Value Flows](https://valueflo.ws))
-
-### Products
-
-- [Cobuy](https://cobuy.nz): group wholesale purchases as a consumer co-op
-
-### Publications
-
-- ["A new financial model for consulting"](https://medium.com/enspiral-tales/a-new-financial-model-for-consulting-c7781661a1ec#.9cybojkj7)
-
-## About this handbook
-
-[![Stories in Ready](https://badge.waffle.io/enspiral-root-systems/handbook.svg?label=ready&title=Ready)](https://waffle.io/enspiral-root-systems/meta)
+* [Media and Publications](media/readme.md)
+* [Context](context/readme.md)
```

Comment:
i reckon we should provide easy links directly to the pages within context, since they provide a high-level overview of who we are.